lib/rails/generators/anycable/setup/local/local_generator.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module AnyCableRailsGenerators
+  module Setup
+    # Generator to help set up AnyCable locally
+    class LocalGenerator < ::Rails::Generators::Base
+      SERVER_VERSION = "v0.6.4"
+      OS_NAMES = %w[linux darwin freebsd win].freeze
+      CPU_NAMES = %w[amd64 arm64 386 arm].freeze
+
+      namespace "anycable:setup:local"
+      source_root File.expand_path("templates", __dir__)
+
+      def server
+        answer = nil
+
+        until [1, 2, 3].include?(answer.to_i)
+          answer = ask "How do you want to install AnyCable-Go WebSocket server? (1) Skip, (2) Homebrew, (3) Download binary"
+        end
+
+        case answer.to_i
+        when 1
+          say_status :help, "🛈 You can read about how to install AnyCable WebSocket here 👉 https://docs.anycable.io/#/anycable-go/getting_started", :yellow
+          return
+        when 2
+          run "brew install anycable-go", abort_on_failure: true
+          run "anycable-go -v", abort_on_failure: true
+        when 3
+          return unless download_binary
+        else
+          raise ArgumentError, "Unknown answer: #{answer}"
+        end
+
+        say_status :info, "✔ AnyCable-Go WebSocket server has been installed"
+      end
+
+      def configs
+        inside("config") { template "cable.yml" }
+      end
+
+      def cable_url
+        environment(nil, env: :development) do
+          'config.action_cable.url = ENV.fetch("CABLE_URL", "ws://localhost:3334/cable").presence'
+        end
+
+        environment(nil, env: :production) do
+          'config.action_cable.url = ENV["CABLE_URL"].presence'
+        end
+
+        say_status :info, "✔ 'config.action_cable.url' has been configured"
+      end
+
+      def proc_files
+        file_name = "Procfile.dev"
+
+        if File.exist?(file_name)
+          append_file file_name, 'anycable: bundle exec anycable --server-command "anycable-go --port 3334"'
+        else
+          say_status :help, "🛈 We recommend using Hivemind - a process manager tool 👉 https://github.com/DarthSim/hivemind", :yellow
+
+          template file_name if yes? "Do you want to create '#{file_name}'?"
+        end
+      end
+
+      def heroku
```

Comment:
What about https://github.com/anycable/capistrano-anycable?

Maybe, we can add two separate generators: `anycable:setup:deployment:heroku` and  `anycable:setup:deployment:capistrano`?

And shouldn't it be a part of the base `setup` generator?

Reply:
Sure, we can describe all deployment methods that we know =)
But for the first stage, I choose only two methods. Will do in a separate PR.

> part of the base setup generator?

Agree